### PR TITLE
Allow running samples CMakeLists.txt as stand alone

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

File: samples/cpp/speech_sample/CMakeLists.txt [hunk 1]
```diff
@@ -25,14 +25,24 @@ if(NOT TARGET zlib::zlib)
...
+                         "${Samples_BINARY_DIR}/../../thirdparty/zlib" EXCLUDE_FROM_ALL)
     endif()
 endif()
 
 if(EXISTS "${Samples_SOURCE_DIR}/thirdparty/cnpy")
+    # OpenVINO package puts thirdparty to samples dir
     add_subdirectory("${Samples_SOURCE_DIR}/thirdparty/cnpy"
                      "${Samples_BINARY_DIR}/thirdparty/cnpy" EXCLUDE_FROM_ALL)
+elseif(EXISTS "${Samples_SOURCE_DIR}/../../thirdparty/cnpy" AND NOT TARGET cnpy)
+    # Allow running samples CMakeLists.txt as stand alone from openvino sources
```
Comment (ilya-lavrenov) [inline samples/cpp/speech_sample/CMakeLists.txt:43]: we need to add guard that cnpy target does not exist

Comment (Wovchena) [reply samples/cpp/speech_sample/CMakeLists.txt:43]: I'm not sure what this means. Isn't `NOT TARGET cnpy` enough?

Comment (ilya-lavrenov) [reply samples/cpp/speech_sample/CMakeLists.txt:43]: Yes, I'm talking about it

Comment (Wovchena) [reply samples/cpp/speech_sample/CMakeLists.txt:43]: So, no action required?

**Pull request merged**